Assets/Character/Scripts/Run.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Run : MonoBehaviour {
+
+    public float moveSpeedX;
+    private float moveDirX;
+
+    Rigidbody2D rb;
+	// Use this for initialization
+	void Start () {
+
+        rb = GetComponent<Rigidbody2D>();
+	}
+
+    // Update is called once per frame
+    void Update()
+    {
+        moveDirX = Input.GetAxis("Horizontal");
+    }
+	void FixedUpdate () {
+       
+        rb.velocity = new Vector2(moveDirX * moveSpeedX, rb.velocity.y);
+	    
```

Comment:
So basically...

void FixedUpdate() 
{
    float dirX = Input.getAxis...
    rb.addForce //if you decide to use addForce
}